tests/phpunit/integration/Modules/Analytics_4Test.php

```diff
@@ -2923,6 +2923,52 @@ public function test_create_custom_dimension() {
 		$this->assertEquals( "/v1beta/properties/$property_id/customDimensions", $request_url['path'] );
 	}
 
+	public function test_create_custom_dimension_without_optional_params() {
```

Comment:
This test name is a bit confusing when compared with the existing `test_create_custom_dimension__required_params()`:

https://github.com/google/site-kit-wp/blob/66c60822f669d225af698f1fb4aad769695a2e63/tests/phpunit/integration/Modules/Analytics_4Test.php#L2781

`test_create_custom_dimension__required_params()` is testing the required parameters for the whole request whereas this test is validating the requirements for the custom dimension.

I'd suggest renaming along these lines:

```suggestion
	public function test_create_custom_dimension__without_optional_fields() {
```